liquibase-standard/src/main/java/liquibase/snapshot/jvm/ForeignKeySnapshotGenerator.java

```diff
@@ -224,6 +224,10 @@ protected ForeignKeyConstraintType convertToForeignKeyConstraintType(Integer jdb
             if (jdbcType == DatabaseMetaData.importedKeyCascade) {
                 return ForeignKeyConstraintType.importedKeyCascade;
             } else if (jdbcType == DatabaseMetaData.importedKeyNoAction) {
+                if (database instanceof SybaseASADatabase) {
```

Comment:
Seems we are missing an import here:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project liquibase-standard: Compilation failure
Error:  /home/runner/work/liquibase/liquibase/liquibase-standard/src/main/java/liquibase/snapshot/jvm/ForeignKeySnapshotGenerator.java:[227,41] cannot find symbol
Error:    symbol:   class SybaseASADatabase
Error:    location: class liquibase.snapshot.jvm.ForeignKeySnapshotGenerator
```

Reply:
Good catch! This is fixed in https://github.com/liquibase/liquibase/pull/4531/commits/7a2824f1e51c4f0e6de7673111d28375feecb17b.

In fact, I missed to rebuild after rebasing my fix on the master branch. As (unfortunately) the Liquibase project is not providing automatic PR builds to let a robot detect this, I have to perform (time-consuming) full builds of each separate bug fix manually, which is a bit annoying. 🙄